hammerblade/torch/tests/test_mlp_mnist.py

```diff
@@ -39,7 +38,7 @@ def forward(self, x):
 # Forward pass
 # -------------------------------------------------------------------------
 
-@pytest.mark.skipif(os.environ.get('USE_HB_EMUL') is None, reason="Slow on cosim")
+@pytest.mark.skipif(torch.hb_emul_on, reason="Slow on cosim")
```

Comment:
Really like this!

Reply:
Yes, should avoid using build time env variables at run time.